### DISCUSSION TITLE
Private social network

### DISCUSSION THREAD

**Discussion by SebDePapyrus** [mastodon/mastodon#18158] [General]
Hi, may be my question is dummy, any =way if I do not ask, I will never know... Is it possible to use mastodon to setup a private social network for my company employee?

Thanks,
Best,
Sébastien.

**Comment by ClearlyClaire** [comment]
Yes, look into `LIMITED_FEDERATION_MODE`, which turns off most of the public pages and does not federate with any other server unless you manually allow them first.
